Automattic/woocommerce-payments
# Add tooltips to each balance title in the account balances card

Resolves #5724

### Changes proposed in this Pull Request

- Adds a help icon tooltip to each of the account balance card fund titles.
- When a help icon tooltip is clicked, a tooltip will be rendered.
- ~Updates the `Tooltip` component with an `ignoreMouseHover` props~  
Uses the ClickTooltip component to show the tooltip content only via click interactions.
- Adds documentation URLs to the Account Balances `strings.ts` file.
- Reduces the coupling between BalanceBlock and content, which is now passed down as props (https://github.com/Automattic/woocommerce-payments/pull/5805#discussion_r1145376190).

> **Note**
> ~I am concerned about the~ (**Updated in b1c1b3e3971659c3023cb0b7b74e24f369e433cc**) low contrast of links rendered as tooltip content. I think this should be raised when we request design feedback, since this is a WCPay-wide component.

https://user-images.githubusercontent.com/3147296/230227968-0c846fc1-f363-421c-b703-095de26656c6.mov

**Available funds**
_"The amount of funds available to be deposited. Learn more."_
- 🔗 https://woocommerce.com/document/woocommerce-payments/deposits/deposit-schedule/ 

![8453d6dbe03b1ced521b13531bf93a26](https://user-images.githubusercontent.com/3147296/230228074-c6c42f06-c651-4c25-9566-615f06b40e69.png)


**Negative available funds**
- _"Learn more about why your account balance may be negative."_
- 🔗 https://woocommerce.com/document/woocommerce-payments/fees-and-debits/account-showing-negative-balance

![75004307573fccbd0d491f406cb2a54a](https://user-images.githubusercontent.com/3147296/230228052-9cec3039-e708-46d8-ab9a-0ea629be9461.png)


**Pending funds**
- _"The amount of funds still in the X day pending period. Learn more."_
- ❗ Note: The X will need to vary based on the merchant country and their pending period. 
- 🔗 https://woocommerce.com/document/woocommerce-payments/deposits/deposit-schedule/

![5b6aa9d183a37b03246ab091c2f23256](https://user-images.githubusercontent.com/3147296/230228024-6b129e6f-7009-4414-98a9-ddbdb1e57a03.png)

### Testing instructions





* Enable the `_wcpay_feature_simplify_deposits_ui` feature flag.
  To enable, run `update_option( '_wcpay_feature_simplify_deposits_ui', '1' );` in [WP Console](https://wordpress.org/plugins/wp-console/).
* Navigate to Payments → Overview.
* View each tooltip by clicking it. Hovering should not trigger the tooltip to be displayed.
* Ensure the correct text and documentation link is rendered, as listed above.
* To test the negative account balance tooltip, make a test purchase with a disputed test card number (`4000000000000259`).

-------------------


**Post merge**
